service/controller/azureconfig/handler/vnetpeering/resource.go

```diff
@@ -0,0 +1,60 @@
+package vnetpeering
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/azure-operator/v5/client"
+)
+
+const (
+	Name = "vnetpeering"
+)
+
+type Config struct {
+	ClientFactory          client.OrganizationFactory
+	CPAzureClientSet       *client.AzureClientSet
+	HostResourceGroup      string
+	HostVirtualNetworkName string
```

Comment:
While here, should these be changed to `ManagementCluster` prefix (from `Host`)?

Reply:
renamed